policy/lib/tekton/trusted.rego

```diff
@@ -90,7 +90,6 @@ data_errors contains error if {
 					"required": ["effective_on", "ref"],
 					"additionalProperties": false,
 				},
-				"uniqueItems": true,
```

Comment:
Remove the `with_severity_for_pattern` call from below? Or maybe also revert this as part of this PR: https://github.com/enterprise-contract/ec-policies/pull/1225

Reply:
We should do this as a cleanup, but let's go ahead and merge it as is.